### ISSUE TITLE
Memory Paranoia Bug

### ISSUE THREAD

**Issue by Hollz100** [Wynncraft/Issues#8215]
The bug is explained a little more thoroughly on the forums (https://forums.wynncraft.com/threads/having-several-issues-with-memory-paranoia.189924/) but in simpler terms, here's what happens:

After defeating Mr. Caritat's remains and trying to use the letter (dropped item) to pass through the rift, the rift kept pushing me back and the book was telling me I still needed to defeat him. 7 defeats later, I finally get through (one of the letters is removed from my inventory) and it spawns me back in the tutorial area, forcing me to go through the tutorial again.

Video of the battles: https://www.youtube.com/watch?v=seWTcG7VgGI
Screenshot of the game making me repeat the tutorial: https://prnt.sc/fb8riq

I did the tutorial and went back to the mansion to speak to the guy outside just in case but am having to go through all the quest's steps once again to get back to where I was.

**Comment by ittnoraA** [comment]
The rift is an NPC which you need to right click with a letter in your inventory to be able to pass through

**Comment by Hollz100** [comment]
Yes, I did that. Tbh the problem I'm reporting here is really the fact that I got tp'ed back to the tutorial, not that it wasn't letting me through the rift ;) 

In any case - Some times I clicked, sometimes I just walked through, because I wasn't sure which I was meant to do. But I did still try that.

**Comment by LordHelmchen98** [comment]
What mc version did you use back then?

**Comment by Hollz100** [comment]
It will have been Labymod 1.8.

**Comment by LordHelmchen98** [comment]
Weird 1.8 should work fine as 1.8(.9) should be the most stable version for Wynn. Anyway the tutorial area is kind of a backup if tp stuff goes wrong. That might be the reason why you ended up there. Also we're you able to complete the quest now?

**Comment by Hollz100** [comment]
I had to do the tutorial again then go back to the house - then it makes you start a few a quest objectives back. But I did manage to get through fine afterwards.
